releng/org.eclipse.emfcloud.modelserver.glsp.releng.target/targetdefinition.tpd

```diff
@@ -27,7 +27,7 @@ location "https://download.eclipse.org/elk/updates/releases/0.7.1/" {
 }
 
 location "https://download.eclipse.org/emfcloud/emfjson-jackson/p2/nightly/1.3/1.3.1.202109010859" {
```

Comment:
shouldn't we update this to the 2.0 release?

Reply:
The Model Server is not updated to the 2.0 release yet and still has the [1.3.1,2.0.0) constraint. We would need to update the Model Server first. I will do so for the next change that comes around.